carbon-design-system/carbon-addons-iot-react
# fix(tablecellrenderer): add min width for truncation

Closes #3798

**Summary**

- Prevent TableHeader cell with tooltip from extreme truncation

**Change List (commits, features, bugs, etc)**

- Add min-width 40px for cells with tooltip
- Update cypress image snapshot

**Acceptance Test (how to verify the PR)**

- Go to [this story](https://deploy-preview-3799--carbon-addons-iot-react.netlify.app/?path=/story/1-watson-iot-table--playground)
- In "Data rendering" tab set wrapCellText to auto, in "Column configuration" tab enable "Demo column tooltips"
- Resize screen horizontally-
- See that content is truncated but min width is set to 40px

**Regression Test (how to make sure this PR doesn't break old functionality)**

- tests here



**Things to look for during review**
